store/store_test.go

```diff
@@ -23,8 +23,8 @@ type testData struct {
 func testConfig() *Config {
 	return &Config{
 		Path:               util.TempDirPath(),
-		TxCacheSize:        1024,
-		SortitionCacheSize: 1024,
+		TxCacheWindow:      1024,
```

Comment:
Old name is explicit for here, I think size better as window.

Reply:
This cache keeps transactions if they are not older than 8,640 blocks. We don't have a fixed size but a fixed window.
The old name was not accurate.